avm/utilities/pipelines/platform/Set-AvmGitHubIssueOwnerConfig.ps1

```diff
@@ -42,10 +42,6 @@ function Set-AvmGitHubIssueOwnerConfig {
   if ($issue.title.StartsWith('[AVM Module Issue]')) {
     $moduleName = ($issue.body.Split("`n") -match 'avm/(?:res|ptn)')[0].Trim().Replace(' ', '')
 
-    if ([string]::IsNullOrEmpty($moduleName)) {
```

Comment:
Why removing this?

Reply:
@eriqua  Máté asked for it, as it is already covered by a different logic (you cannot select a non-existing module).

Reply:
How can we make sure this is run only on the specific issue template where the condition is met by default? I guess it would be enough to create an issue with title starting with `[AVM Module Issue]` to enter this condition, isn't that the case?

Reply:
@eriqua  Good point. I reverted it back.